ArtOfIllusion/src/artofillusion/RecentFiles.java

```diff
@@ -26,15 +26,21 @@ Changes copyright (C) 2017 by Maksim Khramov
   private static final int MAX_RECENT = 10;
 
   /** Given a BMenu, fill it in with a list of items for recent files. */
-
+  public static BMenu createRecentMenu()
+  {
+      BMenu rfm = Translate.menu("openRecent");
+      createMenu(rfm);
+      return rfm;
+  }
+  
   public static void createMenu(BMenu menu)
   {
     menu.removeAll();
     Preferences pref = Preferences.userNodeForPackage(RecentFiles.class);
-    String recent[] = pref.get("recentFiles", "").split(File.pathSeparator);
-    for (int i = 0; i < recent.length; i++)
+    String recentFiles[] = pref.get("recentFiles", "").split(File.pathSeparator);
+    for (String recent: recentFiles)
     {
-      final File file = new File(recent[i]);
+      final File file = new File(recent);
```

Comment:
This set of changes seems to have nothing to do with the main theme of this PR.

In addition, you are adding a public member to the API. What is the purpose here? Just trying to save a couple of lines of code in one very basic instance?

Reply:
Fixed this